src/main/java/com/team4/testingsystem/entities/CoachGrade.java

```diff
@@ -0,0 +1,92 @@
+package com.team4.testingsystem.entities;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "coach_grade")
+public class CoachGrade {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "grade")
+    private Integer grade;
+
+    @ManyToOne
+    @JoinColumn(name = "question_id")
+    private Question question;
+
+    @ManyToOne
+    @JoinColumn(name = "test_id")
+    private Test test;
+
+    public CoachGrade() {
+    }
+
+    public CoachGrade(Integer grade, Question question, Test test) {
```

Comment:
Minor:
constructor fields order seems a bit strange.
In a test, for a particular question, we are marking a grade. So I'll propose to use that order.